aten/src/ATen/native/native_functions.yaml

```diff
@@ -2996,22 +2996,22 @@
   matches_jit_signature: True
   variants: function, method
   dispatch:
-    AffineCPU: dequantize_
-    PerChannelAffineCPU: dequantize_
+    AffineCPU: dequantize_quant
+    PerChannelAffineCPU: dequantize_quant
 
 - func: q_scale(Tensor self) -> Scalar
   matches_jit_signature: True
   variants: function, method
   dispatch:
-    AffineCPU: q_scale_
-    PerChannelAffineCPU: q_scale_
+    AffineCPU: q_scale_quant
+    PerChannelAffineCPU: q_scale_quant
```

Comment:
For `PerChannelAffine` quantization, this `q_scale` still returns a scalar?

Reply:
Oh, I'll remove PerChannelAffineCPU, we should have another API for PerChannel quantization.